string command
int16 iteration_num